web/src/components/storage/ConfigEditorMenu.tsx

```diff
@@ -102,6 +104,13 @@ export default function ConfigEditorMenu() {
             {_("Configure DASD")}
           </DropdownItem>
         )}
+        <DropdownItem
+          key="reprobe-link"
+          onClick={reactivate}
+          description={_("Update the list of available disks and activate crypt devices")}
```

Comment:
NP: I wonder if something three words shorter would work


```suggestion
          description={_("Update available disks and activate crypt devices")}
```